practice/tic-tac-toe/coords_board.py

```diff
@@ -0,0 +1,166 @@
+"""Class Module for Tic-Tac-Toe game that handles the board."""
+
+
+class CoordsTTTBoard:
```

Comment:
Try going through this whole class also and having the computer code out patterns.